### PR TITLE
chore: declare missing commands

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
some commands are registered but not declared

### How to test this PR?

Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

File: extensions/kind/package.json [hunk 1]
```diff
@@ -81,7 +81,17 @@
...
       ]
-    }
+    },
+    "commands": [
+      {
+        "command": "kind.install",
+        "title": "Kind: Install..."
+      },
+      {
+        "command": "kind.image.move",
```
Comment (jeffmaury) [inline extensions/kind/package.json:91]: This command is already declared in line 71 ?

Comment (benoitf) [reply extensions/kind/package.json:91]: @jeffmaury the command is bound to a menu line 71 but it's still not declared

**Comment by benoitf** [comment]
@jeffmaury or @cdrage ?

**Review by cdrage — APPROVED** [review]
This looks correct, weird that it's bound to menu but we have to re-declare it. Today I learned something new haha.

LGTM!

**Pull request merged**